packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js

```diff
@@ -264,6 +270,20 @@ export default class Overlay {
         width: this.tipBoundsWindow.innerWidth,
       },
     );
+
+    this.rects.forEach(rect => {
+      if (!rect.node.matches(':popover-open')) {
+        // $FlowFixMe[prop-missing]: Flow doesn't recognize Popover API
+        // $FlowFixMe[incompatible-use]: Flow doesn't recognize Popover API
+        rect.node.showPopover();
+      }
+    });
+
+    if (!this.tip.tip.matches(':popover-open')) {
+      // $FlowFixMe[prop-missing]: Flow doesn't recognize Popover API
+      // $FlowFixMe[incompatible-use]: Flow doesn't recognize Popover API
+      this.tip.tip.showPopover();
+    }
```

Comment:
I thought hidePopover was unnecessary in this case because the element is completely removed from the DOM during the remove phase.